static/app/views/alerts/rules/issue/setupMessagingIntegrationButton.tsx

```diff
@@ -79,6 +74,11 @@ function SetupMessagingIntegrationButton({
     return null;
   }
 
+  trackAnalytics('onboarding.messaging_integration_setup_button_shown', {
+    organization,
+    ...analyticsParams,
+  });
+
```

Comment:
we only create a log if the button is shown, we don't have a `shown: false` log if the button is not shown

Reply:
We want to be able to answer these questions:
1. Are the people who see the button install integration?
2. Are the people who see the button create projects the same amount as the ones who don't?

I think we can answer the first one for sure with these logs. We can't answer 2 though because the other logs doesn't say if they actually saw the button, it only tells us if the feature flag is on. So I'm thinking an improvement could be to logs there in `createProject.tsx` if they actually saw the button instead of feature flag.

Otherwise when you create the funnel the next step of funnel has more people in it than the step before. I think another way to do this is to it on Amplitude and define that step of funnel as saw the `onboarding.messaging_integration_setup_button_shown` AND `project_creation_page.created` but I'm not sure about it. Need to try it on Amplitude.

Reply:
I think it's difficult to add to the `createProject` logs if the button was shown or not, because this will only log if the button was being shown at the time that the project was created. This wouldn't account for if the button was shown when the page was opened, the user installed an integration, the button went away, and then the project was created.

Reply:
What if we compared these 2 cases:

- `project_creation_page.created` with no feature flag
- `onboarding.messaging_integration_setup_button_shown` AND `project_creation.page.created`

Would this answer the 2nd question?

Reply:
Yes, I believe the AND works. I just wasn't sure if we can AND on amplitude. Thank you.